### PR TITLE
fix(print): #433 - print certificate with multiple pages

### PR DESCRIPTION
**Pull request by Nebulis**
This PR faces printing issues as explained in #438 

In order to test can use the file attached [Business.txt](https://github.com/OpenCerts/opencerts-website/files/3792547/Business.txt)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "integration:watch": "nodemon --exec \"npm run integration\" --watch src"
   },
   "dependencies": {
+    "@govtechsg/decentralized-renderer-react-components": "1.1.2",
```
Comment (Nebulis) [inline package.json]: remove penpal

File: src/reducers/certificate.js [hunk 2]
```diff
@@ -42,10 +42,7 @@ export const initialState = {
 
   encryptedCertificate: {},
   encryptedCertificateState: states.INITIAL,
-  encryptedCertificateError: null,
```
Comment (Nebulis) [inline src/reducers/certificate.js]: I forgot to delete some actions => CERTIFICATE_TEMPLATE_REGISTER

**Comment by rjchow** [comment]
fix the printing from iframe at some point thanks

**Comment by Nebulis** [comment]
one day sure

**Pull request merged**